optaplanner-docs/src/main/asciidoc/Chapter-Planner_introduction/Chapter-Planner_introduction.adoc

```diff
@@ -119,6 +115,84 @@ OptaPlanner supports several optimization algorithms to efficiently wade through
 Depending on the use case, some optimization algorithms perform better than others, but it's impossible to tell in advance. **With OptaPlanner, it is easy to switch the optimization algorithm**, by changing the solver configuration in a few lines of XML or code.
 
 
+[[requirements]]
+== Requirements
+
+OptaPlanner is _open source_ software, released under http://www.apache.org/licenses/LICENSE-2.0.html[the Apache Software License 2.0].
+This license is very liberal and allows reuse for commercial purposes.
+Read http://www.apache.org/foundation/licence-FAQ.html#WhatDoesItMEAN[the layman's explanation].
+
+OptaPlanner is 100% pure Java^TM^
+ and runs on any JVM 1.6 or higher.
+It <<integration,integrates very easily>> with other Java^TM^
+ technologies.
```

Comment:
This newline happened due to the asciidoc migration. I fix them when I come across them, feel free to do the same.
so "technologies." should be on the same line as the line above.